frontend/src/lib/components/pages/home/post.svelte

```diff
@@ -19,6 +19,12 @@
 	};
 
 	let props: Props = $props();
+
+	// Correctly stop the propagation and prevent default behavior
+	function stopEventPropagation(e: Event) {
+		e.stopPropagation();
+		e.preventDefault();
+	}
```

Comment:
https://svelte.dev/docs/svelte/v5-migration-guide#Event-changes-Event-modifiers
should be able to pass functions, also move it to lib/functions